src/Nethermind/Nethermind.Serialization.Rlp/TxDecoder.cs

```diff
@@ -314,7 +314,7 @@ private void EncodeShardBlobPayloadWithoutPayload(T item, RlpStream stream, RlpB
             stream.Encode(item.Value);
             stream.Encode(item.Data);
             _accessListDecoder.Encode(stream, item.AccessList, rlpBehaviors);
-            if(item.MaxFeePerDataGas.Value == 42_000_000_000)
+            if (item.MaxFeePerDataGas.Value == 42_000_000_000)
```

Comment:
Oh I've almost forgot that we break the client, let's think how we can do the same without adding bugs

Reply:
Rolled back